data.sql

```diff
@@ -0,0 +1,11 @@
+vet_clinic=# INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+              VALUES ('Agumon', '2020-02-03', 10.23, TRUE, 0);
+
+vet_clinic=# INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+              VALUES ('GABUMON', '2018-11-15', 8.0, TRUE, 2);
+
+vet_clinic=# INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+              VALUES ('Pikachu', '2021-01-07', 15.04, FALSE, 1);
+
+vet_clinic=# INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+               VALUES ('Devimon', '2017-05-12', 11.0, TRUE, 5);
```

Comment:
- [ ] Kindly remove **vet_clinic=#** cuz these are the words which your terminal has. But they are not part of your query so kindly remove them :blush: